microsoftgraph/microsoft-graph-devx-api
# [Fix] Gets missing Service principal scopes in GetAllPermissions

Fixes https://github.com/microsoftgraph/microsoft-graph-devx-api/issues/645

## Overview
- Adds in (missing) scopes information details from the Service Principal after aggregating all scopes from the permissions-mapping file.
- Updates tests appropriately to validate the above fix.

## Testing Instructions
- Run: https://localhost:44399/permissions
- Search for: 
  - RoleManagementPolicy.Read.Directory
  - RoleManagementPolicy.ReadWrite.Directory
  - RoleEligibilitySchedule.Read.Directory
  - RoleEligibilitySchedule.ReadWrite.Directory
  - RoleAssignmentSchedule.Read.Directory
  - RoleAssignmentSchedule.ReadWrite.Directory

You should be able to see these permission from the list.
